src/doc/src/reference/rust-version.md

```diff
@@ -0,0 +1,154 @@
+# Rust Version
+
+The `rust-version` field is an optional key that tells cargo what version of the
+Rust toolchain you support for your package.
+
+```toml
+[package]
+# ...
+rust-version = "1.56"
+```
+
+The Rust version must be a bare version number with at least one component; it
+cannot include semver operators or pre-release identifiers. Compiler pre-release
+identifiers such as -nightly will be ignored while checking the Rust version.
+
+> **MSRV:** Respected as of 1.56
+
+## Uses
+
+**Diagnostics:**
+
+When your package is compiled on an unsupported toolchain,
+Cargo will provide clearer diagnostics than reporting invalid syntax or missing functionality in `std`.
```

Comment:
```suggestion
Cargo will provide clearer diagnostics about the insufficient toolchain version rather than reporting invalid syntax or missing functionality in the standard library.
```